src/VisualStudio/LiveShare/Impl/Client/RoslynLSPClientService.cs

```diff
@@ -91,22 +91,22 @@ public Task<ICollaborationService> CreateServiceAsync(CollaborationSession colla
                     JObject.FromObject(new ServerCapabilities
                     {
                         // Uses Roslyn client.
-                        DocumentOnTypeFormattingProvider = new DocumentOnTypeFormattingOptions(),
-                        DocumentRangeFormattingProvider = true,
-                        DocumentFormattingProvider = true,
-                        DocumentSymbolProvider = true,
                         CodeActionProvider = true,
                         ExecuteCommandProvider = new ExecuteCommandOptions(),
-                        DocumentHighlightProvider = true,
-                        ReferencesProvider = true,
-                        DefinitionProvider = true,
-                        SignatureHelpProvider = new SignatureHelpOptions() { },
-                        CompletionProvider = new CompletionOptions(),
-                        ImplementationProvider = true,
 
                         // Uses LSP SDK client.
+                        CompletionProvider = null,
```

Comment:
Is the null/false the default? If so, want to just delete these so all that is left is the stuff that's still there? (The diff makes it a bit confusing but I spent some time staring at this going "what's left?"...)